OJP/OJP_Places.xsd

```diff
@@ -203,10 +203,23 @@
 	</xs:group>
 	<xs:simpleType name="PlaceTypeEnumeration">
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="stop"/>
+			<xs:enumeration value="stop">
+				<xs:annotation>
+					<xs:documentation>stop covers StopPlace and StopPoint.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
 			<xs:enumeration value="address"/>
 			<xs:enumeration value="poi"/>
-			<xs:enumeration value="coord"/>
+			<xs:enumeration value="coord">
+				<xs:annotation>
+					<xs:documentation>deprecated. Is replaced by geoPosition.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="location">
+				<xs:annotation>
+					<xs:documentation>New value for geoPosition.</xs:documentation>
```

Comment:
What about `Geographic position consisting of coordinates. Replaces the deprecated value coord.`

Reply:
fixed